lib/active_model/serializer.rb

```diff
@@ -54,10 +57,11 @@ def self.type(type)
     end
 
     def self.attributes(*attrs)
+      options = attrs.last.class == Hash ? attrs.pop : {}
       attrs = attrs.first if attrs.first.class == Array
 
       attrs.each do |attr|
-        attribute(attr)
+        attribute(attr, options)
```

Comment:
@bf4 I'm not sure I understand why we would not want to support options for `attributes`. Could you elaborate?